draft-janfred-radext-radiusdtls-bis.md

```diff
@@ -178,25 +179,27 @@ While the client may be able to deduce the operational state of the local server
 As a result, when a NAS does not receive a response to a request, this could be the result of packet loss between the NAS and proxy, a problem on the proxy, loss between the RADIUS proxy and server, or a problem with the server.
 
-When UDP is used as a transport protocol, the absence of a reply can cause a client to deduce (incorrectly) that the proxy is unavailable.
+The absence of a reply can cause a client to deduce (incorrectly) that the proxy is unavailable.
 The client could then fail over to another server or conclude that no "live" servers are available (OKAY state in {{!RFC3539}}, Appendix A).
 This situation is made even worse when requests are sent through a proxy to multiple destinations.
 Failures in one destination may result in service outages for other destinations, if the client erroneously believes that the proxy is unresponsive.
 
-For RADIUS/TLS, it is RECOMMENDED that implementations utilize the existence of a TCP connection along with the application-layer watchdog defined in {{RFC3539}}, Section 3.4 to determine that the server is "live".
-RADIUS/TLS clients MUST mark a connection DOWN, if the network stack indicates that the connection is no longer active.
-If the network stack indicates that the connection is still active, clients MUST NOT decide that it is down until the application-layer watchdog algorithm has marked it DOWN.
-RADIUS/TLS clients MUST NOT decide that a RADIUS/TLS server is unresponsive until all TLS connections to it have been marked down.[^contradiction]{:jf}
+It is REQUIRED that implementations utilize the existence of a TCP/DTLS connection along with the application-layer watchdog defined in {{RFC3539, Section 3.4}} to determine the liveliness of the server.
 
-[^contradiction]: The specification in RFC6613 is contradictory here. Section 2.4 says that it is recommended to have a watchdog. Section 2.6 says it must be used.
+RADIUS/(D)TLS clients MUST mark a connection DOWN if one or more of the following conditions are met:
+* The administrator has marked the connection administrative DOWN.
+* The network stack indicates that the connection is no longer viable.
+* The application-layer watchdog algorithm has marked it DOWN.
 
-The above requirements do not forbid the practice of a client proactively closing connections or marking a server as DOWN due to an administrative decision.
+If a RADIUS/(D)TLS client has multiple connection to a server, it MUST NOT decide to mark the whole server as DOWN until all connections to it have been marked DOWN.
 
-It is RECOMMENDED that RADIUS/(D)TLS nodes implement the Status-Server extension as described in {{?RFC5997}} to detect the liveness of the peer without dependence on successful authentications.
+It is REQUIRED that RADIUS/(D)TLS clients implement the Status-Server extension as described in {{!RFC5997}} as the application level watchdog to detect the liveliness of the peer in the absence of responses.
```

Comment:
Add the requirement to use Status-Server. Since RFC5997 is an informational RFC and we have to mark this as normative reference, this would lead to a downref. But should be ok.